src/superphot_plus/mlp.py

```diff
@@ -514,7 +553,11 @@ def classify_from_fit_params(self, fit_params):
             Sums to 1 along each row.
         """
         fit_params_2d = np.atleast_2d(fit_params)  # cast to 2D if only 1 light curve
-        test_features, _, _ = normalize_features(fit_params_2d, MEANS_TRAINED_MODEL, STDDEVS_TRAINED_MODEL)
```

Comment:
Can we still import these constants for the testing methods using ModelConfig? ([logs](https://github.com/lincc-frameworks/superphot-plus/actions/runs/5858382241/job/15882227094?pr=108#step:5:161))

Reply:
Just fixed test cases to use filler values for normed means and stddevs. Classifier needs to be rerun on test data so there's associated normalization values to reinstitute the expected_classes check though.

Reply:
Apart from that the PR looks good!